.github/workflows/pr-validation.yml

```diff
@@ -23,4 +23,11 @@ jobs:
       - name: Run static analysis
         run: ./vendor/bin/phpstan
       - name: Run tests
-        run: ./vendor/bin/phpunit --coverage-text
+        run: ./vendor/bin/phpunit --coverage-clover=coverage.xml
+      - name: Fix code coverage paths
+        run: sed -i 's@'$GITHUB_WORKSPACE'@/github/workspace/@g' coverage.xml
+      - name: SonarCloud Scan
```

Comment:
you'll need a condition to run the analysis on a single version of PHP to avoid erratic behaviour

Reply:
Done